ote_sdk/ote_sdk/tests/entities/test_color.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import pytest
+
+from ote_sdk.entities.color import Color, ColorEntity
+
+from ote_sdk.tests.constants.ote_sdk_components import OteSdkComponent
+from ote_sdk.tests.constants.requirements import Requirements
+
+red = 40
+red_hex = "28"
+green = 210
+green_hex = "d2"
+blue = 43
+blue_hex = "2b"
+alpha = 255
+alpha_hex = "ff"
+color_hex = f"{red_hex}{green_hex}{blue_hex}"
+
+color = Color.from_hex_str(color_hex)
+
+@pytest.mark.components(OteSdkComponent.OTE_SDK)
+class TestColor:
+
+    @pytest.mark.priority_medium
+    @pytest.mark.component
+    @pytest.mark.reqids(Requirements.REQ_1)
+    def test_color(self):
+        """
+        <b>Description:</b>
+        Check that Color can correctly return the value
+
+        <b>Expected results:</b>
+        Test passes if the results matches
```

Comment:
`results match`, please find all such entries and fix

Reply:
Typos fixed